Areas/Admin/ParkingZoneController.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using Parking_Zone.Data;
+using Parking_Zone.Models;
+
+namespace Parking_Zone.Areas.Admin
+{
+    [Area("Admin")]
+    public class ParkingZoneController : Controller
+    {
+        private readonly ApplicationDbContext _context;
+
+        public ParkingZoneController(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        // GET: Admin/ParkingZones
+        public async Task<IActionResult> Index()
+        {
+            return View(await _context.ParkingZones.ToListAsync());
+        }
+
+        // GET: Admin/ParkingZones/Details/5
+        public async Task<IActionResult> Details(Guid? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var parkingZone = await _context.ParkingZones
+                .FirstOrDefaultAsync(m => m.Id == id);
+            if (parkingZone == null)
+            {
+                return NotFound();
+            }
+
+            return View(parkingZone);
+        }
+
+        // GET: Admin/ParkingZones/Create
+        public IActionResult Create()
+        {
+            return View();
+        }
+
+        // POST: Admin/ParkingZones/Create
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
+        // For more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create([Bind("Id,Name,Address,DateOfEstablishment")] ParkingZone parkingZone)
```

Comment:
Bind is not necessary

```suggestion
        public async Task<IActionResult> Create(ParkingZone parkingZone)
```